src/components/Heading/Heading.mdx

```diff
@@ -52,23 +54,149 @@ proper order and not skipping any. For example, the following is invalid HTML:
   language="html"
 />
 
+When either the overline or subtitle elements are rendered as part of the
+heading, then the whole lockup will be wrapped in an `<hgroup>` element with
+proper `role` and `aria-roledescription` attributes applied.
+
+<Source
+  code={`
+<hgroup role="group" aria-roledescription="Heading group">
+  <h2>Heading Title</h2>
+  <p aria-roledescription="subtitle">Subtitle</p>
+</hgroup>
+`}
+  language="html"
+/>
+
 Resources:
 
 - [W3C WAI Headings](https://www.w3.org/WAI/tutorials/page-structure/headings/)
 - [A11y Project Accessible heading structure](https://www.a11yproject.com/posts/how-to-accessible-heading-structure/)
 - [Chakra UI Heading](https://chakra-ui.com/docs/components/typography/heading)
 
-## Default Styles
+## Updated Typographic Styles
+
+New typographic styles were introduced in June 2023. The new styles are the
+recommended options for the default `h` elements and the `size` overrides.
+
+## Native Heading Elements
+
+The `level` prop can be used to set a native heading element (ex. `<h1>`,
+`<h2>`, ...).
+
+The default text styles for the native heading element levels are directly
+related to the corresponding values of the `size` prop.
+
+For example, for the **recommended** options listed below, the `heading1` size
+is the default style for the `<h1>` element. The `heading2` size is the default
+style for the `<h2>` element. And so on.
+
+For the **deprecated** options listed below, the `primary` size is the default
```

Comment:
I think it bit hard to read and find the alternative to the deprecated option. A cleaner way to show the alternatives to the deprecated options is to use a table or even just a list.

Reply:
I was already thinking about moving this text to the "deprecated" section, so that is what I did.

As for the table comparison, I made note of your suggestion and we'll see about something like this for a future update.